xbmc/interfaces/json-rpc/schema/methods.json

```diff
@@ -209,7 +204,6 @@
       "items": {
         "type": "object",
         "properties": {
-          "playercoreid": { "type": "integer", "minimum": 1, "required": true },
           "name": { "$ref": "Global.String.NotEmpty", "required": true },
           "type": { "type": "string", "enum": [ "internal", "external", "remote" ], "required": true },
```

Comment:
If external control from Kodi is shutdown as well as starting then if it's possible to know when it's upnp and the upnp renderer uuid it could allow advanced remotes to recover control.

If there's no way to get uuid then knowing upnp or not is kinda useless.